packages/connector-wechat-native/src/index.test.ts

```diff
@@ -57,15 +57,25 @@ describe('getAccessToken', () => {
     expect(openid).toEqual('openid');
   });
 
-  it('throws SocialAuthCodeInvalid error if accessToken not found in response', async () => {
+  it('throws SocialAuthCodeInvalid error if errcode is 40029', async () => {
     nock(accessTokenEndpointUrl.origin)
       .get(accessTokenEndpointUrl.pathname)
       .query(parameters)
-      .reply(200, {});
+      .reply(200, { errcode: 40_029, errmsg: 'invalid code' });
```

Comment:
just curious: `errorcode` and `errmsg`, are those WeChat reserved keys? neither snake case nor camel case. looks so weird.

Reply:
They are WeChat API reserved keys for error messages.